modelclone/admin.py

```diff
@@ -107,7 +107,18 @@ def clone_view(self, request, object_id, form_url='', extra_context=None):
                 prefixes[prefix] = prefixes.get(prefix, 0) + 1
                 if prefixes[prefix] != 1 or not prefix:
                     prefix = "%s-%s" % (prefix, prefixes[prefix])
-                formset = FormSet(data=request.POST, files=request.FILES,
+
+                request_files = request.FILES
+                filter_params = {'%s__pk' % original_obj.__class__.__name__.lower(): original_obj.pk}
+                inlined_objs = inline.model.objects.filter(**filter_params)
+                for n, inlined_obj in enumerate(inlined_objs.all()):
+                    for field in inlined_obj._meta.fields:
+                        if isinstance(field, FileField) and field not in request_files:
+                            value = field.value_from_object(inlined_obj)
+                            file_field_name = '{}-{}-{}'.format(prefix, n, field.name)
+                            request_files[file_field_name] = value
```

Comment:
Thanks @hsoft!
I just found one issue using it with the `sampleproject` locally, here is how to reproduce:

1. Create a `Post` with an image, let's say `image1.jpg`
2. Click Clone
3. Before saving the duplicate, select a new image, like `image2.jpg`, and Save
4. The new duplicated post will still have `image1.jpg`

I think if you use `setdefault()` in `request_files` here should solve it

Reply:
I'll add a test case for this, probably tomorrow.